tests/ops/test_channel_ops.py

```diff
@@ -433,7 +433,8 @@ def test_p_arbitrary(self, p, tol):
         expected_K1 = np.sqrt(p) * Z
         assert np.allclose(op(p, wires=0).kraus_matrices()[1], expected_K1, atol=tol, rtol=0)
 
-    @pytest.mark.parametrize("angle", np.linspace(0, 2 * np.pi, 7))
+    # !TODO: bring back angle 0 when the bug fixed https://github.com/PennyLaneAI/pennylane/pull/6684#issuecomment-2552123064
+    @pytest.mark.parametrize("angle", np.linspace(0, 2 * np.pi, 7)[1:])
```

Comment:
Can you add a shortcut story to the top of the backlog for this TODO and include the story ID in this TODO comment?
```suggestion
    # TODO: bring back angle 0 when the bug fixed https://github.com/PennyLaneAI/pennylane/pull/6684#issuecomment-2552123064
    @pytest.mark.parametrize("angle", np.linspace(0, 2 * np.pi, 7)[1:])
```